core-build/test-web-library-build/gulpfile.js

```diff
@@ -13,4 +13,10 @@ build.setConfig({
 
 build.preCopy.cleanMatch = ['src/preCopyTest.ts'];
 
+build.karma.enabled = false;
```

Comment:
Are we using karma and jest together?

Reply:
No.

---
In reply to: [145591969](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145591969) [](ancestors = 145591969)